### PR TITLE
chore(docs): add bug and feature request templates (#41)

### PR DESCRIPTION
**Pull request by Villanuevand**
Add basic version of: 
- Bug issue template.
- Feature Request template.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/02-feature-request.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+name: Feature Request
+description: Request a new or improve a feature in Analogjs
```
Comment (brandonroberts) [inline .github/ISSUE_TEMPLATE/02-feature-request.yml:2]:
```suggestion
description: Request a new or improved feature in Analog
```

**Review by brandonroberts — CHANGES_REQUESTED** [review]
Looking good! A few changes in each file

File: .github/ISSUE_TEMPLATE/01-bug-report.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
+name: Bug Report
+description: Report a bug in Analogjs
+labels: [bug]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for help us to grow. ❤️
```
[the remark this answers is not in the document]
Comment (Villanuevand) [reply .github/ISSUE_TEMPLATE/01-bug-report.yml]: Change added!

File: .github/ISSUE_TEMPLATE/01-bug-report.yml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    attributes:
+      label: Please provide the environment you discovered this bug in.
+      description: Use the [Analog StackBlitz example](https://stackblitz.com/github/analogjs/analog/tree/main/packages/create-analog/template-angular-v14?file=vite.config.ts&preset=node) to create a reproduction.
+    validations:
+      required: true
+
+  - type: dropdown
+    id: type_bug
+    attributes:
+      label: Which scope was the bug found it
```
[the remark this answers is not in the document]
Comment (Villanuevand) [reply .github/ISSUE_TEMPLATE/01-bug-report.yml]: Change added!

File: .github/ISSUE_TEMPLATE/01-bug-report.yml [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+      options:
+        - create-analog
+        - vite-plugin-angular
+        - Docs
+        - Don't known / other
```
[the remark this answers is not in the document]
Comment (Villanuevand) [reply .github/ISSUE_TEMPLATE/01-bug-report.yml]: Change added!

File: .github/ISSUE_TEMPLATE/02-feature-request.yml (hunk 1 above)
Comment (Villanuevand) [reply .github/ISSUE_TEMPLATE/02-feature-request.yml]: Change added!

File: .github/ISSUE_TEMPLATE/01-bug-report.yml [hunk 5]
```diff
@@ -0,0 +1,58 @@
+name: Bug Report
+description: Report a bug in Analog
+labels: [bug]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for helping us grow. ❤️
+        If do you like Analogjs enough please consider support through [sponsoring](https://github.com/sponsors/brandonroberts) 💖
```
Comment (brandonroberts) [inline .github/ISSUE_TEMPLATE/01-bug-report.yml:10]:
```suggestion
        If do you like Analog enough please consider support through [sponsoring](https://github.com/sponsors/brandonroberts) 💖
```

File: .github/ISSUE_TEMPLATE/02-feature-request.yml [hunk 6]
```diff
@@ -0,0 +1,45 @@
+name: Feature Request
+description: Request a new or improved a feature in Analog
+labels: [enhancement]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for helping us grow. ❤️
+        If do you like Analogjs enough please consider support through [sponsoring](https://github.com/sponsors/brandonroberts) 💖
```
Comment (brandonroberts) [inline .github/ISSUE_TEMPLATE/02-feature-request.yml:10]:
```suggestion
        If do you like Analog enough please consider support through [sponsoring](https://github.com/sponsors/brandonroberts) 💖
```

File: .github/ISSUE_TEMPLATE/02-feature-request.yml [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+        - Docs
+        - Don't known / other
+    validations:
+      required: true
+
+  - type: textarea
+    id: information
+    attributes:
+      label: Information
+      description: Tell us what you want to be added to Analogjs
```
Comment (brandonroberts) [inline .github/ISSUE_TEMPLATE/02-feature-request.yml:28]:
```suggestion
      description: Tell us what you want to be added to Analog
```

**Review by brandonroberts — APPROVED** [review]
nits

**Comment by brandonroberts** [comment]
Thanks! @allcontributors add @Villanuevand for docs

**Pull request merged**